introduction.md

```diff
@@ -0,0 +1 @@
+My name is Amren Miller and I am learning to code by attending a bootcamp. Nice to meet you all.
```

Comment:
Not really truly sure what my favorite path is yet, I just started using codeschool. But I view HTML and CSS as eating your vegetables; they are necessary but not my favorites. I like scripting languages the most.